es-runtime/src/es-api.ts

```diff
@@ -169,7 +170,7 @@ export interface RibbonBar {
     showRibbonTab: (id: string) => HTMLDivElement | null
     hideRibbonTab: (id: string) => HTMLDivElement | null
     components?: {
-        [key: string]: HTMLElement
```

Comment:
We have users of ES that use Angular or other non-React frameworks.

We do not want to be tied to React in the API. We do want to allow React components to use the API and to be ribbon items, etc. We have examples of that use case.